public/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ramsey Apps 1.0</title>
+    <link rel="apple-touch-icon" sizes="57x57" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-57x57.png">
+    <link rel="apple-touch-icon" sizes="60x60" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-60x60.png">
+    <link rel="apple-touch-icon" sizes="72x72" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-72x72.png">
+    <link rel="apple-touch-icon" sizes="76x76" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-76x76.png">
+    <link rel="apple-touch-icon" sizes="114x114" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-114x114.png">
+    <link rel="apple-touch-icon" sizes="120x120" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-120x120.png">
+    <link rel="apple-touch-icon" sizes="144x144" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/apple-touch-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/favicon-16x16.png">
+    <link rel="icon" type="image/x-icon" href="https://cdn.ramseysolutions.net/cms/sites/ramseysolutions-com/favicons/favicon.ico">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body class="bg-light brush">
+      <!--header-->
```

Comment:
🎸  Comments like this are redundant, as the tag right below it is <header> which is self-documenting.